cmd/livepeer/starter/starter.go

```diff
@@ -1210,7 +1210,7 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 		} else {
 			glog.Warningf("!!! No GPU discovered, using CPU for AIWorker !!!")
 			// Create 2 fake GPU instances, intended for the local non-GPU setup
-			gpus = []string{"emulated-0", "emulated-1"}
+			gpus = []string{"emulated-0"}
```

Comment:
What's this change for? The comment above is also outdated now

Reply:
I think it was just confusing for the standard local setup to have multiple containers.